src/table/utils/styling-utils.js

```diff
@@ -153,6 +152,27 @@ export function getBodyCellStyle(layout, theme) {
   };
 }
 
+/**
+ * Get style for total cell
+ *
+ * @param {Object} totalHeaderStyle
+ * @param {String} totalsPosition
+ * @returns total cell style regarding the total head position
+ */
+export function getTotalsCellStyle(totalHeaderStyle, totalsPosition) {
+  if (totalsPosition === 'noTotals') return false;
```

Comment:
Is this possible?

Reply:
Are you asking about the condition ? If so, yes. There are some conditions that total row is not shown ( when it is set to be none or table has only dimension and so on)

Reply:
this is inconsistent with what your description says. And we would have to have two different return types when converting to typescript. You could just return `totalHeaderStyle` as is insteaad

Reply:
This statement will never be true, since we catch that earlier, so you can remove it